### PR TITLE
Add fluentd docu

### PR DESCRIPTION
**Pull request by vwiessner**
Added documentation forFluentd options. To be merged once Taupage with Fluentd is rolled out.

### CODE REVIEW

File: components/taupage.rst [hunk 1]
```diff
@@ -587,6 +587,151 @@ scalyr_region
...
+---------------
+
+**(optional)**
+
+If set to `true` the Fluentd Agent will be started.
+
+.. NOTE::
+   By default Fluentd Agent will send all logs to scalyr. Use either Fluentd Agent or Scalyr Agent unless you know what you are doing, otherwise you might send logs to scalyr twice.
+
+   All fluentd options mentioned depend on this to be set to `true`.
```
Comment (femueller) [inline components/taupage.rst]: I would propose to update "fluentd' to "Fluentd".

**Review by femueller — APPROVED** [review]

**Comment by femueller** [comment]
Besides the typo I would 👍.

**Comment by vwiessner** [comment]
:+1:

File: components/taupage.rst [hunk 2]
```diff
@@ -542,20 +543,64 @@ And the Agent will follow these logs:
...
 .. NOTE::
-   You can also use AWS KMS to encrypt your Scalyr account key. See in the example above.
+   If a logging section is present in ``senza.yaml`` then ``scalyr_account_key``, ``scalyr_application_log_parser``, ``scalyr_custom_log_parser`` and ``scalyr_region`` entries outside the scope of the logging section will be ignored.
 
-If you provide the Scalyr AccountKey in the .yaml file, the agent of Scaylr will be installed and will follow these logs:
+.. _scalyr_account_key:
 
-  * /var/log/syslog
-  * /var/log/auth.log
-  * /var/log/application.log
```
Comment (christianberg) [inline components/taupage.rst:558]: Is this removed from the documentation (which logs are sent by default)? I don't see it being moved somewhere else.

File: components/taupage.rst [hunk 3]
```diff
@@ -542,20 +543,64 @@ And the Agent will follow these logs:
 
 You can get your Account Key from the Logentries Webinterface under /Account/Profile
 
-
 scalyr_account_key
 ------------------
 
 **(optional)**
 
+Options: see :ref:`scalyr_account_key <scalyr_account_key>` in :ref:`logging <logging>`
```
Comment (christianberg) [inline components/taupage.rst:553]: Should we mark the usage of these options outside of the `logging` section as deprecated?

File: components/taupage.rst (hunk 2 above)
Comment (vwiessner) [reply components/taupage.rst:558]: added back to logging -> log_destination

File: components/taupage.rst (hunk 3 above)
Comment (vwiessner) [reply components/taupage.rst:553]: Marked old scalyr options in main yaml config branch as deprecated.

**Pull request merged**